docs/src/getting_started.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Getting Started Guide
+slug: /getting_started
+---
+
+# Getting Started with Charmil
+
+This guide assumes that you already have a working Go environment, if not then please refer to
+[this page](https://golang.org/doc/install) first.
+
+## Charmil SDK
+
+This is the component responsible for constructing modular, multi-repo Golang CLIs with Cobra while providing a variety of high-level solutions for typical challenges like configuration, internationalization, etc.
+&nbsp;
+
+- ### Installation:
+
+  First use `go get` to install the latest tagged release of the framework.
+  The following command will install the framework along with its dependencies:
+
+  ```bash
+  $ go get -u github.com/aerogear/charmil
+  ```
+
+  Next, import the required package into your project:
+
+  - For the config management package:
+
+    ```go
+    import "github.com/aerogear/charmil/core/config"
+    ```
+
+  - For utils packages (eg. color, localize, logging, etc.):
+
+    ```go
+    import "github.com/aerogear/charmil/core/utils/{name_of_the_required_package}"
+    ```
+
+- ### Usage:
+
+  For usage instructions on Charmil SDK, please refer to the docs [over here](./).
```

Comment:
What is that for? It points to itself..

Reply:
Resolved via commit: https://github.com/aerogear/charmil/pull/241/commits/600be390512957dccce755f217441cf244b7e812